prow/github/client.go

```diff
@@ -2155,6 +2155,34 @@ func (c *Client) ListMilestones(org, repo string) ([]Milestone, error) {
 	return milestones, nil
 }
 
+// ListCommits lists the commits in a pull request.
+//
+// GitHub API docs: https://developer.github.com/v3/pulls/#list-commits-on-a-pull-request
+func (c *Client) ListCommits(org, repo string, number int) ([]RepositoryCommit, error) {
+	c.log("ListCommits", org)
+	if c.fake {
+		return nil, nil
+	}
+	var commits []RepositoryCommit
+	err := c.readPaginatedResultsWithValues(
+		fmt.Sprintf("/repos/%v/%v/pulls/%d/commits", org, repo, number),
+		url.Values{
+			"per_page": []string{"100"},
+		},
+		acceptNone,
+		func() interface{} { // newObj
```

Comment:
`// newObj` is not a super useful comment, this means nothing to me

Reply:
Done